en/docs/integration-guides/usecases/datamapper/read-csv-file-and-transform-to-xml-file.md

```diff
@@ -0,0 +1,601 @@
+    | **Result** | Name of the result variable that stores the read content | `csvRecords` |
+    | **Return Type** | The data type of the returned value | `CSV[]` |
+
+    If the return type list doesn’t appear automatically, click the blue **record selector icon**, expand **Types → CSV**, and select **CSV[]** from your defined records.
+
+  4. Bind the CSV Reader function to a Configurable Path
+
+      After selecting the `io:fileReadCsv` function, the next task is to configure how the file path is supplied.  
+      Rather than hardcoding a literal path, this step dynamically binds the function to a configurable variable (`inputCSV`) defined in your project’s configurations.
+
+      1. Switch the Path Input Mode
+
+        In the `fileReadCsv` configuration pane:
+
+        1. Locate the **Path*** field at the top.
+        2. By default, it shows a **Text** input mode.
+        3. Click **Expression** next to it.  
+            This changes the mode to accept dynamic values such as variables or configurables instead of a static string.
+
+
+      2. Open the Value Picker
+
+        Once in **Expression** mode:
+        - A blue **ƒx** button appears to the left of the input field.  
+        - Click that icon to open the **Value Picker** dropdown.  
+        - The picker shows several categories:
+
+          - `Create Value`
+          - `Inputs`
+          - `Variables`
+          - **Configurables** 
+          - `Functions`
+
+
+      3. From the **Configurables** section, select **`inputCSV`** (this is the configurable variable that stores the file path for the input CSV).
+
+  5. Name the Result Variable
+
+    In the **Result*** field, enter a meaningful variable name to store the data returned by the reader: csvRecords
+    This variable will now hold the array of CSV records returned from `fileReadCsv`.
+
+
+  6. Define the Return Type
+
+    In **Return Type***, click the type selector box and pick from the list of available record types under your **Types** section.
+    This ensures the system interprets each row of the CSV as a structured `CSV` record instead of a generic string map.
+
+
+  7. Save the Configuration
+
+    Click **Save** to insert the configured node into the automation flow.
+
+    You now have a fully functional CSV reader node that:
+
+    - Reads the file from `${inputCSV}`,
+    - Parses its contents into structured `CSV[]` data,
+    - Stores it in the variable `csvRecords`.
+
+The **fileReadCsv** node is added to your Automation diagram, connected directly under **Start**.
+
+This means the Automation will:
+
+1. Start execution.  
+2. Use `fileReadCsv` to read the input file located at the path stored in `${inputCSV}`.  
+3. Store the resulting CSV records in the variable `csvData` for later mapping.
+
+
+> **Tip:** Always ensure the first row of your CSV file includes headers that match the `CSV` record field names. Otherwise, `fileReadCsv` will not correctly map the columns to your CSV record.
```

Comment:
Add tips in the standard way. Refer to info box in https://bi.docs.wso2.com/developer-guides/create-a-project/#steps-to-create-an-integration-project 

As below, instead of info, use the text tip.

<img width="967" height="195" alt="image" src="https://github.com/user-attachments/assets/7ec9596f-99ca-4a99-a05c-4d5be0d790cf" />